mitodl/hugo-course-publisher
# Line break shortcode

#### Pre-Flight checklist


#### What are the relevant tickets?
Related to https://trello.com/c/C3x9nQvq/32-fix-rendering-of-tables-in-ocw-to-hugo-generated-markdown

#### What's this PR do?
Adds a shortcode to the project that allows you to insert semantic line breaks into markdown content using `{{< br >}}` and updates the sample content to reflect this change.

#### How should this be manually tested?
Run the site, line breaks should now properly appear within table cells without having to turn on `unsafe=true`
